docs/tutorials/flow-app-quickstart.mdx

```diff
@@ -68,48 +70,36 @@ config({
 
 > Learn more about [configuring Discovery](../reference/discovery) or [setting configuration values](../reference/api.md#setting-configuration-values).
 
-The main page for Next.js apps is located in `pages/index.js`. So let's finish configuring our dapp by going in the `pages/` directory and importing the config file into the top of our `index.js` file. We'll then swap out the default component in `index.js` to look like this:
+The main page for Next.js apps is located in `pages/index.js`. So you can finish configuring our dapp by going into the `pages/` folder and importing the config file into the top of our `index.js` file. You will import the config file in the default component in `index.js`.
 
 > **Replace file:** `./pages/index.js`
 
 ```jsx
-import "../flow/config";
-
-import Head from 'next/head'
 
-export default function Home() {
-  return (
-    <div>
-      <Head>
-        <title>FCL Quickstart with NextJS</title>
-        <meta name="description" content="My first web3 app on Flow!" />
-        <link rel="icon" href="/favicon.png" />
-      </Head>
-
-      <main>
-        Hello World
-      </main>
-
-    </div>
-  )
-}
+import "../flow/config";
 
 ```
 
-Now we're ready to start talking to Flow!
+Now you are ready to start talking to Flow!
 
 ## Authentication
 
-To authenticate a user, all an app has to do is call `fcl.logIn()`. Sign up and unauthenticate are all also as simple as `fcl.signUp()` and `fcl.unauthenticate()`.  Once authenticated, FCL sets an object called `fcl.currentUser` which exposes methods for watching changes in user data, signing transactions, and more. For more information on the `currentUser`, read more [here](../reference/api.md#current-user).
+In this section, you will learn how to add sign-up and sign-in functionality in your flow dapp.
+To authenticate a user, you will call `fcl.logIn()`, if the user is authenticated, FCL sets an object called `fcl.currentUser`, which exposes methods for watching changes in user data, signing transactions, and more. For more information on the `currentUser`, read more [here](../reference/api.md#current-user).
 
-Let's add in a few buttons for sign up/login and also subscribe to changes on the `currentUser`. When the user is updated (which it will be after authentication), we'll set the user state in our component to reflect this. To demonstrate user authenticated sessions, we'll conditionally render a component based on if the user is or is not logged in.
+For the sake of this tutorial, you will add a few buttons for sign up/log in and subscribe to changes on the `currentUser`. When the user is updated (which it will be after authentication), you will set the user state in our component to reflect this. To demonstrate user authenticated sessions, you will conditionally render a component based on if the user is or is not logged in.
```

Comment:
if you change to 2nd person (you), also change the other pronouns (e.g. "our")

Reply:
> if you change to 2nd person (you), also change the other pronouns (e.g. "our")

I actually did change it, on Grammarly. i will fix it.